### DISCUSSION TITLE
Nested node types

### DISCUSSION THREAD

**Discussion by sepsol** [ianstormtaylor/slate#4447] [Ideas]
I'm using this library called [`plate`](https://github.com/udecode/plate) which, as you know, is built on top off of `slate`. I want to implement nested `blockquote` feature similar to GitHub's Quote Reply feature in my app but it doesn't seem to be possible with the current API. Below is a demonstration of what I'm trying to achieve along with its markdown. It would be great if this feature could be added to slate.

> This is a blockquote.
> This is the second line in that blockquote.
> > This is a nested blockquote.
> > This is the next line in the nested blockquote.
> > * This is a bullet list in the blockquote.
> > - [ ] This is a checkbox in a nested blockquote.

```
> This is a blockquote.
> This is the second line in that blockquote.
> > This is a nested blockquote.
> > This is the next line in the nested blockquote.
> > * This is a bullet list in the blockquote.
> > [] This is a checkbox in a nested blockquote which is disabled.
```

**Comment by alessiostalla** [comment]
I'm interested too. The documentation talks about complex nested structure, but all examples are flat or, at most, a block-level element (paragraph) containing an inline element (link).

For context, I'm experimenting with Slate to build a structured editor, where some parts of the text are fixed (keywords) and others are variable. And I need to nest such components.
